### PR TITLE
Provide an example for Dubbo load balancing

### PR DESCRIPTION
**Pull request by complone**
Dubbo-based random load balancing, add demonstration examples

### CODE REVIEW

File: dubbo-sample-gateway/src/main/java/org/apache/dubbo/samples/CallbackListenerImpl.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.apache.dubbo.samples;
+
+
+import org.apache.dubbo.samples.callback.api.CallbackListener;
+
+/**
+ * @author complone
```
Comment (beiwei30) [inline dubbo-sample-gateway/src/main/java/org/apache/dubbo/samples/CallbackListenerImpl.java]: pls. remove author info, use English for comment, and add apache license header.

File: dubbo-sample-gateway/pom.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.11.1</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (beiwei30) [inline dubbo-sample-gateway/pom.xml]: pls. refer to other samples to add integration test.

**Comment by complone** [comment]
Just saw that these days are busy not on github

**Comment by complone** [comment]
> @complone could you pls. take a look at my review comment?

I have removed files that do not meet the project requirements

**Pull request merged**